geo-types/Cargo.toml

```diff
@@ -11,13 +11,23 @@ description = "Geospatial primitive data types"
 edition = "2018"
 
 [features]
-relative_eq = []
+use-rstar = ["rstar", "approx"]
 
 [dependencies]
-approx = "0.4.0"
+approx = { version = "0.4.0", optional = true }
```

Comment:
There seemed to be some concern about introducing the new approx impls be default. I believe this is because people want to keep geo-types as small as possible, since it's a dependency used by many libs.

Rather than introducing a new `relative_eq` feature, I think those impls would be more logically gated on the inclusion of `approx` which is itself now optional.